src/test/java/org/gridsuite/gateway/GatewayTest.java

```diff
@@ -0,0 +1,55 @@
+package org.gridsuite.gateway;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.cloud.contract.wiremock.AutoConfigureWireMock;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.reactive.server.WebTestClient;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+
```

Comment:
Is it possible to use the same testing techniques as our other microservices ? mockmvc, mockbean etc ? Ideally the test would not start a webserver at all.